src/org/loklak/api/search/QuoraProfileScraper.java

```diff
@@ -53,18 +53,20 @@ public QuoraProfileScraper() {
         super();
```

Comment:
Can the changes to QuoraProfileScraper be moved to a separate PR as this PR as the name suggests deals with configuring GithubProfileScraper.